src/main/java/Database.java

```diff
@@ -0,0 +1,75 @@
+import java.util.ArrayList;
+
+public class Database {
+    public static ArrayList<Task> taskList = new ArrayList<>();
+
+    public static void markDone(String arg) {
+        try{
+            int index = Integer.parseInt(arg);
+            taskList.get(index - 1).markAsDone();
+            markDoneResponse(index);
+        } catch(NumberFormatException e) {
+            System.out.println("The command is not recognised.");
+        } catch (IndexOutOfBoundsException e){
+            System.out.println("The index is out of range. This action is aborted.");
```

Comment:
Please ensure consistencies of whether to leave a space after 'catch' before adding the catch conditions.